src/background_thread.c

```diff
@@ -857,12 +864,12 @@ background_thread_boot1(tsdn_t *tsdn) {
 	}
 
 	background_thread_info = (background_thread_info_t *)base_alloc(tsdn,
-	    b0get(), ncpus * sizeof(background_thread_info_t), CACHELINE);
+	    b0get(), max_background_threads * sizeof(background_thread_info_t), CACHELINE);
```

Comment:
since this is allocated at boot, I don't think we can increase the number beyond this?